.travisci/README.md

```diff
@@ -0,0 +1,44 @@
+# Continuous Integration Support for Travis
+
+This directory contains files and helpers for Travis-CI.
+
+## Toolchains
+
+The [`toolchains.xml`](toolchains.xml) file contains Maven toolchains descriptions
+for the installations found on the Travis build images.  They may need to be updated
+as changes are made to the Travis images.
+
+## Uploading Build Reports to GCS
+
+The .travis.yml is configured to upload build reports and test
+artifacts, like SWTBot `screenshots/` and `surefire-reports/`, to
+a GCS bucket.  The bucket can be accessed at:
+
+   https://console.cloud.google.com/storage/browser/BUCKET
+
+Configuring the uploading requires creating a GCS bucket, and
+creating a service account with the _Storage Object Creator_ role.
+
+### Configuring Auto-Deletion for the GCS Bucket
+
+`gcs.lifecycle.json` provides an [auto-deletion configuration
+policy](https://cloud.google.com/storage/docs/managing-lifecycles#delete_an_object)
+suitable for a GCS bucket hosting the build reports.  It should be
+installed with:
+
+```
+$ gsutil lifecycle set .travisci/gcs.lifecycle.json gs://BUCKET
+```
+
+### Configuring the Service Account Key
+
+  1. Create a key for the service account from the [Cloud
+     Console](https://console.cloud.google.com/iam-admin/serviceaccounts)
+     with role _Storage > Storage Object Creator_.
```

Comment:
I think you also need to verify that "Enable G Suite Domain-wide Delegation" is checked, though it should be by default.

Reply:
Don't we want it *unchecked*?  Does this service need to access user's data?

> In enterprise applications you may want to programmatically access a user's data without any manual authorization on their part. In G Suite domains, the domain administrator can grant third-party applications with domain-wide access to its users' data — this is referred as domain-wide delegation of authority. To delegate authority this way, domain administrators can use service accounts with OAuth 2.0.

Reply:
I read that as, if unchecked then a user will have to go through an oAuth screen before access is granted. I.e. a pure service account is not enough. User action is required.

I could be misreading it.

Reply:
I verified that my service account has this option unchecked.  Pantheon shows the following text:

![screen shot 2017-12-15 at 3 07 38 pm](https://user-images.githubusercontent.com/202851/34058672-e49cf4ce-e1a9-11e7-8e47-4c45b33cf9b9.PNG)